Hardware/Nic/NicGroup.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.NetworkInformation;
+using System.Text;
+
+namespace OpenHardwareMonitor.Hardware.Nic
+{
+    internal class NicGroup : IGroup
+    {
+
+        private List<Hardware> hardware = new List<Hardware>();
+        private NetworkInterface[] nicArr;
+
+        public NicGroup(ISettings settings)
+        {
+            int p = (int)Environment.OSVersion.Platform;
+            if ((p == 4) || (p == 128))
+            {
+                hardware = new List<Hardware>();
+                return;
+            }
+            nicArr = NetworkInterface.GetAllNetworkInterfaces();
+            for (int i = 0; i < nicArr.Length; i++)
+            {
+                if (nicArr[i].NetworkInterfaceType.ToString() == "Ethernet")
```

Comment:
This should be compared with the enum (NetworkInterfaceType) instead of string, and this also limits it to Ethernet only types, which makes wireless for example useless. I think it can monitor everything, except for Loopback, Tunnel and Unknown.